src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfIndividualElements = sourceString.split(';');
```

Comment:
[CODE STYLE] - The variable name `arrayOfIndividualElements` is not descriptive enough and does not reflect the contents of the array. Consider a name that describes the data it holds, such as `styleDeclarations` or `cssRules`, which indicates that each element represents a CSS rule or declaration.